gulpfile.js

```diff
@@ -36,13 +36,21 @@ gulp.task('ext:lint', () => {
 });
 
 // Copy icons for OE
-gulp.task('ext:copy-assets', (done) => {
+gulp.task('ext:copy-OE-assets', (done) => {
     return gulp.src([
         config.paths.project.root + '/src/objectExplorer/objectTypes/*'
     ])
     .pipe(gulp.dest('out/src/objectExplorer/objectTypes'));
 });
 
+// Copy icons for Query History
+gulp.task('ext:copy-queryHistory-assets', (done) => {
+    return gulp.src([
```

Comment:
I wonder if it would be better to have a `copy-icons` or `copy-resources` task instead, so when we have more icons\resources we don't have to keep adding tasks.  Then instead of putting icons under query history folder there could be a `media` folder or something that help all this type of asset.  It may work better as the extension grows?

Reply:
Let me address this in a separate PR.